pages/common/~.md

```diff
@@ -0,0 +1,16 @@
+# Tilde
```

Comment:
I've been using capitalized names for these symbol based pages ever since they were suggested to me when I started doing these. https://github.com/tldr-pages/tldr/pull/10548
I also like the capitalization as it sets the symbol based pages apart from actual commands.